build/import/Packages.targets

```diff
@@ -11,7 +11,7 @@
       https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json;
       https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json;
       https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json;
-      https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json;
+      https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json
```

Comment:
Any reason to remove the semi-colon? Just makes it easier to add more sources in the future if it remains.

Reply:
No, that's just an artifact of changing the sources a couple times.